mautic/mautic
# [UX] Improve the comparison between the two form type options

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
Currently, the comparison between the two form type options might not provide sufficient information for users to make an informed choice. This could potentially lead to users not fully understanding the differences and benefits of each option.

The pull request proposes to improve the comparison by providing more detailed and specific information about each form type option. This change aims to provide users with a clearer understanding of each option, helping them to make a choice that best suits their needs.

Users who frequently interact with the form creation process stand to benefit from this change. It enhances the clarity of the user interface and contributes to a more seamless user experience.

![image](https://github.com/mautic/mautic/assets/116097999/af194155-765b-4ecb-a2de-36c40d642c2c)

To fix layout issues, I migrated a .less file from the bootstrap package into CoreBundle



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Forms
3. Create a new form
4. See the new list of differences
